src/simpleSchema.ts

```diff
@@ -0,0 +1,51 @@
+import { Dictionary, expandUrl, loadField, unionOfNoneTypeOrStrtype, LoadingOptions, Saveable, ValidationException } from './util/internal'
+
+export class Simple_schema extends Saveable {
+  loadingOptions: LoadingOptions
+  label?: string
+  extensionFields?: Dictionary<any>
+
+  constructor (loadingOptions: LoadingOptions, label?: string, extensionFields?: Dictionary<any>) {
+    super()
+    this.loadingOptions = loadingOptions ?? new LoadingOptions({})
+    this.label = label
+    this.extensionFields = extensionFields ?? {}
+  }
+
+  static override async fromDoc (doc: any, baseuri: string, loadingOptions: LoadingOptions, docRoot?: string): Promise<Saveable> {
+    const _doc = Object.assign({}, doc)
+
+    const errors: ValidationException[] = []
+    if ('label' in _doc) {
+      try {
+        var label: string | undefined = await loadField(_doc.label, unionOfNoneTypeOrStrtype, baseuri, loadingOptions)
```

Comment:
@ZimmerA should we use this `label` declared in this scope when creating a new `Simple_schema` further down this function? Or is it just confirming we can load the field?

Reply:
Already fixed a couple of days ago, this was just weird